ros2/test.bzl

```diff
@@ -29,6 +30,15 @@ def ros2_test(name, nodes, launch_file, deps = None, data = None, **kwargs):
         tags = tags,
     )
 
+    deps = deps or []
+    deps.append(ament_setup_target)
```

Comment:
Please also handle here `data` as well. Then you don't need to handle that later in _ros2_launch_pytest_test nor _ros2_launch_testing_test.